docs/current_docs/cookbook/snippets/clone-git-repository/php/src/MyModule.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Container;
+
+use function Dagger\dag;
+
+#[DaggerObject]
+class MyModule
+{
+    #[DaggerFunction]
+    public function clone(string $repository, string $branch): Container
```

Comment:
I know the PHP SDK doesn't support enums so we can't reproduce the other snippets _exactly_ but it may be worth trying to get as close as possible: maybe add a $locator argument and a switch-case for three possible values, or default to branch if none match. WDYT?

Reply:
Yeah, we should definitely keep that inline. I'll update the example to align.